packages/backend-next/src/index.ts

```diff
@@ -17,16 +17,12 @@
 import { createBackend } from '@backstage/backend-defaults';
 
 const backend = createBackend();
+backend.add(import('@backstage/plugin-discovery-backend'));
 
 backend.add(import('@backstage/plugin-adr-backend'));
 backend.add(import('@backstage/plugin-app-backend/alpha'));
 backend.add(import('@backstage/plugin-azure-devops-backend'));
 backend.add(import('@backstage/plugin-badges-backend'));
-backend.add(import('@backstage/plugin-catalog-backend-module-unprocessed'));
-backend.add(
-  import('@backstage/plugin-catalog-backend-module-scaffolder-entity-model'),
-);
-backend.add(import('@backstage/plugin-catalog-backend/alpha'));
```

Comment:
These removals are for demonstrating the split backend working, they will be restored after this PR is reviewed.